src/components/calcite-tabs/calcite-tabs.scss

```diff
@@ -1,21 +1,24 @@
 :host {
-  display: flex;
-  flex-direction: column;
+  @apply flex flex-col;
 }
 
 :host([position="below"]) {
-  flex-direction: column-reverse;
+  @apply flex-col-reverse;
 }
 
 section {
-  display: flex;
-  flex-grow: 1;
-  overflow: hidden;
-  border-top: 1px solid var(--calcite-ui-border-1);
+  @apply flex
+    flex-grow
+    overflow-hidden
+    border-t
+    border-t-solid
```

Comment:
`border: solid` is/will be set via `@tailwind base` so this will not be needed in the future.